tests/unit/AccountContactMissingDialog.spec.ts

```diff
@@ -38,19 +38,16 @@ describe('AccountContactMissingDialog.vue', () => {
       propsData: { dialog: true }
     })
 
-    // Override window.location.href
-    let redirectedUrl = ''
-    // eslint-disable-next-line accessor-pairs
-    Object.defineProperty(window.location, 'href', {
-      set (url: string) {
-        redirectedUrl = url
```

Comment:
This test was failing after moving from happy-dom to jsdom:
![cannot redefine property](https://github.com/bcgov/business-create-ui/assets/122301442/6df12fc0-6cb8-4c50-a02c-987163c94ca2)

The new code fixes it. Reference: https://github.com/jestjs/jest/issues/5124

Reply:
Nice find!